consensus/block_validation.go

```diff
@@ -49,16 +50,14 @@ func (ebc *DefaultBlockValidationClock) EpochSeconds() uint64 {
 
 // DefaultBlockValidator implements the BlockValidator interface.
 type DefaultBlockValidator struct {
-	clock     BlockValidationClock
```

Comment:
I believe we can replace the `BlockValidationClock` with the `BlockClock` interface once it matures a bit more (in the coming block validation PR)